doc/nrf/introduction.rst

```diff
@@ -36,6 +36,42 @@ Every |NCS| release consists of a combination of all included repositories at di
 See the :ref:`repos_and_revs` section for a comprehensive list of repositories and their current revisions.
 The revision of each of those repositories is determined by the current revision of the main (manifest) repository ``sdk-nrf``.
 
+Versions and revisions
+**********************
+
+The |NCS| uses a versioning scheme similar to `Semantic versioning`_, but with
+important semantic differences.  Every release of the |NCS| is identified with
+a version string, in the format ``MAJOR.MINOR.PATCH``. The version numbers are
+incremented based on the following criteria:
+
+* The ``MAJOR`` version number is increased seldom, whenever a release is deemed
+  to be introducing a large number of substantial changes across the board
+* The ``MINOR`` version number is increased every time a major release is cut. Major
+  releases are the default |NCS| release kind. They introduce functionality and
+  may break APIs in certain cases
+* The ``PATCH`` version number is increased whenever a minor or bugfix release
+  is cut. Minor releases are those that only address functional issues but do
+  not introduce functionality
+
+In between releases, |NCS| is not static. Instead it changes its revision every
+time a Git commit is merged into the `sdk-nrf`_ repository. We consider the
+revision of the SDK to be equivalent to the ``sdk-nrf``'s repository revision
+because it is the :ref:`manifest repository <zephyr:west-manifests>`. This means
+that, by virtue of containing the `west manifest file`_, its revision uniquely
+identifies the revisions of all other repos included in the SDK.
```

Comment:
```suggestion
In between releases, |NCS| is not static.
Instead, it changes its revision every time a Git commit is merged into the `sdk-nrf`_ repository.
The revision of the SDK is considered to be equivalent to the ``sdk-nrf``'s repository revision, because it is the :ref:`manifest repository <zephyr:west-manifests>`.
This means that, by virtue of containing the `west manifest file`_, its revision uniquely identifies the revisions of all other repositories included in the SDK.
```